code/game/gamemodes/objective.dm

```diff
@@ -23,18 +23,9 @@
 /datum/objective/proc/get_target()
 	return target
 
-
-/datum/objective/proc/get_crewmember_minds()
-	. = list()
-	for(var/V in data_core.locked)
-		var/datum/data/record/R = V
-		var/mob/M = R.fields["reference"]
-		if(M && M.mind)
-			. += M
-
 /datum/objective/proc/find_target()
 	var/list/possible_targets = list()
-	for(var/datum/mind/possible_target in get_crewmember_minds())
```

Comment:
This really should of been used for latejoin only as the roundstart antags ARE picked first.